apps/web/src/hooks/__tests__/useTouchDevice.test.ts

```
@@ -0,0 +1,119 @@
+    }),
+    addListener: vi.fn(),
+    removeListener: vi.fn(),
+    onchange: null,
+    dispatchEvent: vi.fn(),
+  };
+
+  return {
+    mql,
+    triggerChange: (newMatches: boolean) => {
+      mql.matches = newMatches;
+      listeners.forEach((l) => l());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)